src/app/app-routing.module.ts

```diff
@@ -1,11 +0,0 @@
-import { NgModule } from '@angular/core';
-import { Routes, RouterModule } from '@angular/router';
-
-
-const routes: Routes = [];
-
-@NgModule({
-  imports: [RouterModule.forRoot(routes)],
-  exports: [RouterModule]
-})
```

Comment:
Don't we need this file ? Why it is deleted ?

Reply:
it's moved to another file called routes.ts which serves the same purpose.